assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -97,58 +101,61 @@ export class CounterFrontend extends Component {
 
     let style = this.props.style || 'plain'; // Needed to convert existing blocks
-    const styleClass = getStyleFromClassName(className);
-    if (styleClass) {
+    const styleClass = getStyleFromClassName( className );
+    if ( styleClass ) {
       style = styleClass;
     }
     const arcLength = 31.5;
 
-    const percent = Math.min(target > 0 ? Math.round(completed / target * 100) : 0, 100);
+    const percent = Math.min( target > 0 ? Math.round( completed / target * 100 ) : 0, 100 );
 
-    let counterClassName = `block counter-block counter-style-${style} ${className ?? ''}`;
-    if (isEditing) counterClassName += ` editing`;
+    let counterClassName = `block counter-block counter-style-${ style } ${ className ?? '' }`;
+    if ( isEditing ) {
+      counterClassName += ' editing';
+    }
 
     return (
       <Fragment>
-        <section className={counterClassName}>
-          {title && !isEditing &&
-            <header>
-              <h2 className="page-section-header">{title}</h2>
-            </header>
+        <section className={ counterClassName }>
+          { title && ! isEditing &&
+					<header>
+					  <h2 className="page-section-header">{ title }</h2>
+					</header>
           }
-          {description && !isEditing &&
-            <p className="page-section-description" dangerouslySetInnerHTML={{ __html: description }} />
+          { description && ! isEditing &&
+					<p className="page-section-description" dangerouslySetInnerHTML={ { __html: description } } />
           }
           <div className="content-counter">
-            {(style === 'bar' || style === 'en-forms-bar') &&
-              <div className="progress-container">
-                <div className={`progress-bar ${style === 'en-forms-bar' ? 'enform-progress-bar' : ''}`} style={{ width: `calc(${percent}% + 20px)` }} />
-              </div>
+            { ( style === 'bar' || style === 'en-forms-bar' ) &&
+						<div className="progress-container">
+						  <div className={ `progress-bar ${ style === 'en-forms-bar' ? 'enform-progress-bar' : '' }` } style={ { width: `calc(${ percent }% + 20px)` } } />
+						</div>
             }
-            {style === 'arc' &&
-              <svg className="progress-arc" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 24 14">
-                <path className="background" d="M 2 12 A 1 1 0 1 1 22 12" />
-                <path className="foreground" d="M 2 12 A 1 1 0 1 1 22 12"
-                  strokeDasharray={arcLength}
-                  strokeDashoffset={`${(1 - percent / 100) * arcLength}`} />
-              </svg>
+            { style === 'arc' &&
+						<svg className="progress-arc" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 24 14">
+						  <path className="background" d="M 2 12 A 1 1 0 1 1 22 12" />
+						  <path className="foreground" d="M 2 12 A 1 1 0 1 1 22 12"
+						    strokeDasharray={ arcLength }
+						    // eslint-disable-next-line no-mixed-operators
```

Comment:
Couldn't we add parenthesis here instead of disabling the rule? 🤔 
```
${ ( 1 - ( percent / 100 ) ) * arcLength }
```

Reply:
Done.